packages/dcos-integration-test/extra/test_helpers.py

```diff
@@ -0,0 +1,3 @@
+from pkgpanda.util import load_json
+
+dcos_config = load_json('/opt/mesosphere/etc/expanded.config.json')
```

Comment:
This brings an external dependency. (Existence of file).
- How do we ensure that the file always exists?
- Is there any reasonable defaults for the test module that can be used in the absence of the file?